.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+# This is a continuous action workflow for bazel, using bazelisk.
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push
+# events but only for the master branch. Triggers on all pull requests.
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
```

Comment:
"build" might not be the most apt name, since this also runs tests.

Reply:
Good point. I updated the name to holistically describe the check.